nuklear_gamepad.h

```diff
@@ -322,9 +380,6 @@ NK_API nk_bool nk_gamepad_init_with_source(struct nk_gamepads* gamepads, struct
 
     // Set the default state for all gamepads.
     for (int i = 0; i < NK_GAMEPAD_MAX; i++) {
-        // Available
-        gamepads->gamepads[i].available = nk_true;
```

Comment:
There is an API change here... The gamepads all now start as unavailable. Need to call `nk_gamepad_set_available()` to enable them.